src/SpectatorView.Native/SpectatorView.Compositor/dependencies.props

```diff
@@ -4,15 +4,15 @@
   <ImportGroup Label="PropertySheets" />
   <PropertyGroup Label="UserMacros">
     <!-- From: https://www.blackmagicdesign.com/support -->
-    <DeckLink_inc>$(SolutionDir)\..\..\external\Blackmagic DeckLink SDK 10.9.11\Win\include</DeckLink_inc>
+    <DeckLink_inc>C:\Blackmagic DeckLink SDK 10.9.11\Win\include</DeckLink_inc>
```

Comment:
><DeckLink_inc>C:\Blackmagic DeckLink SDK 10.9.11\Win\include</DeckLink_inc> [](start = 4, length = 75)

These changes look unintentional?

Reply:
They are intentional. We no longer require these SDKs to be relocated to the project for scripted building. Now we just default to the best expected location for these sdks.

Previously, we had hardcoded dll look up locations in build scripts. Now we use whatever a developer specifies here to resolve the location of these dependencies.